app/src/main/java/com/framgia/fsalon/screen/info/InfoUserPresenter.java

```diff
@@ -0,0 +1,100 @@
+package com.framgia.fsalon.screen.info;
+
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports